flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/container/RedisCommandsContainerBuilder.java

```diff
@@ -66,10 +66,20 @@ public static RedisCommandsContainer build(FlinkJedisPoolConfig jedisPoolConfig)
         genericObjectPoolConfig.setMaxIdle(jedisPoolConfig.getMaxIdle());
         genericObjectPoolConfig.setMaxTotal(jedisPoolConfig.getMaxTotal());
         genericObjectPoolConfig.setMinIdle(jedisPoolConfig.getMinIdle());
+        genericObjectPoolConfig.setTestOnBorrow(jedisPoolConfig.getTestOnBorrow());
+        genericObjectPoolConfig.setTestOnReturn(jedisPoolConfig.getTestOnReturn());
+
+        if (jedisPoolConfig.getTestWhileIdle()) {
+            // default parameters from redis.clients.jedis.JedisPoolConfig
+            genericObjectPoolConfig.setTestWhileIdle(true);
```

Comment:
if this are the default params, why we have to set them?

Reply:
OK, I will write a test to check this. 

As for the default params, it is provided by JedisPoolConfig (a subclass of GenericObjectPoolConfig in Jedis), but not GenericObjectPoolConfig itself, so I chose to set them explicitly. Do you think we should just replace it with JedisPoolConfig when `testWhileIdle` is set to true?

Reply:
it is better to create a variable in the class and have the value defined in only 1 place...
if this value can be obtained in some way from the default values of the library, it is better (always with the constant in the class)